[2021-06-27 07:23:45,063] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: postgretomysql.export_postgresql_customer 2021-06-27T07:23:39.796148+00:00 [queued]>
[2021-06-27 07:23:45,091] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: postgretomysql.export_postgresql_customer 2021-06-27T07:23:39.796148+00:00 [queued]>
[2021-06-27 07:23:45,092] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-06-27 07:23:45,092] {taskinstance.py:1068} INFO - Starting attempt 1 of 2
[2021-06-27 07:23:45,095] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-06-27 07:23:45,113] {taskinstance.py:1087} INFO - Executing <Task(PythonOperator): export_postgresql_customer> on 2021-06-27T07:23:39.796148+00:00
[2021-06-27 07:23:45,121] {standard_task_runner.py:52} INFO - Started process 5472 to run task
[2021-06-27 07:23:45,130] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'postgretomysql', 'export_postgresql_customer', '2021-06-27T07:23:39.796148+00:00', '--job-id', '51', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/postgresql-collect.py', '--cfg-path', '/tmp/tmp3tcyfvzv', '--error-file', '/tmp/tmpg05gcsgu']
[2021-06-27 07:23:45,131] {standard_task_runner.py:77} INFO - Job 51: Subtask export_postgresql_customer
[2021-06-27 07:23:45,191] {logging_mixin.py:104} INFO - Running <TaskInstance: postgretomysql.export_postgresql_customer 2021-06-27T07:23:39.796148+00:00 [running]> on host b69deda3a989
[2021-06-27 07:23:45,273] {taskinstance.py:1282} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=tech@innospark.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=postgretomysql
AIRFLOW_CTX_TASK_ID=export_postgresql_customer
AIRFLOW_CTX_EXECUTION_DATE=2021-06-27T07:23:39.796148+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-06-27T07:23:39.796148+00:00
[2021-06-27 07:23:45,275] {logging_mixin.py:104} INFO - Entering export_postgresql_to_csv SELECT * FROM test.customer
[2021-06-27 07:23:45,295] {base.py:78} INFO - Using connection to: id: docker-postgres. Host: 192.168.1.27, Port: 5432, Schema: , Login: postgres, Password: ***, extra: {}
[2021-06-27 07:23:49,083] {base.py:78} INFO - Using connection to: id: docker-postgres. Host: 192.168.1.27, Port: 5432, Schema: , Login: postgres, Password: ***, extra: {}
[2021-06-27 07:23:49,106] {python.py:151} INFO - Done. Returned value was: None
[2021-06-27 07:23:49,119] {taskinstance.py:1191} INFO - Marking task as SUCCESS. dag_id=postgretomysql, task_id=export_postgresql_customer, execution_date=20210627T072339, start_date=20210627T072345, end_date=20210627T072349
[2021-06-27 07:23:49,164] {taskinstance.py:1245} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-06-27 07:23:49,211] {local_task_job.py:151} INFO - Task exited with return code 0
